lms/envs/production.py

```diff
@@ -1071,3 +1071,6 @@ def get_env_setting(setting):
 
 ############################# CELERY ############################
 CELERY_IMPORTS.extend(ENV_TOKENS.get('CELERY_EXTRA_IMPORTS', []))
+
+############################# MFE API CONFIG CACHE TIMEOUT############################
+MFE_API_CONFIG_CACHE_TIMEOUT = ENV_TOKENS.get('MFE_API_CONFIG_CACHE_TIMEOUT', MFE_API_CONFIG_CACHE_TIMEOUT)
```

Comment:
This is not needed, if the config file has this value, it will get picked up automatically.

This bit of code ensures that: https://github.com/openedx/edx-platform/blob/cb1e8f70a162fc3e181c5a589fd60b60520d652c/lms/envs/production.py#L91

There's a bunch of old lines like this that need to be cleaned up but you don't need to add more.

Reply:
Thanks Feanil, I already made the change 🙌